stack.yaml

```diff
@@ -1,5 +1,6 @@
 resolver: lts-13.19
 extra-deps:
-- galois-field-0.4.0
+- git: git@github.com:adjoint-io/galois-field.git
```

Comment:
This depends on a git version bump to galois-field? This will break Hackage if I upload it.

Reply:
I was using this for `pairing` - will restore 0.4.0.

Reply:
All good now.